gym/wrappers/step_compatibility.py

```diff
@@ -0,0 +1,76 @@
+import gym
+from gym import logger
+
+
+class StepCompatibility(gym.Wrapper):
+    def __init__(self, env, return_two_dones=False):
+        super().__init__(env)
+        self._return_two_dones = return_two_dones
+        if not self._return_two_dones:
+            logger.deprecation(
+                "[StepAPI] Initializing environment in old step API which returns one bool instead of two. "
```

Comment:
The [StepAPI] thing implies there's a syntax like that in gym overall, which is not the case as far as I know, so I think it can be removed